src/RequestClientWrapper.ts

```diff
@@ -327,4 +335,33 @@ export default class RequestClientWrapper {
       return stringValue.startsWith(number.toString());
     }
   }
+
+  private isItJwtToken(token: string): boolean {
+    try {
+      JSON.parse(Buffer.from(token.split('.')[1], 'base64').toString());
+    } catch (e) {
+      return false;
+    }
+
+    return true;
+  }
+
+  private async refreshSupabaseSession(): Promise<string> {
+    const supabase = createClient(SUPABASE_URL, SUPABASE_ANON_KEY);
+    const { data, error } = await supabase.auth.setSession({
+      access_token: this.requestClientWrapperConfiguration.session,
+      refresh_token: this.requestClientWrapperConfiguration.session
+    });
+    if (data && data.session) {
+      const { access_token } = data.session;
+      if (access_token) {
```

Comment:
you don't need this if here. You are already assuming that data.session is populated by your above check. So just return

```
return data.session
```